javascript/features/death_match/death_match_commands.js

```diff
@@ -30,6 +33,10 @@ export class DeathMatchCommands {
         this.manager_.leave(player);
     }
 
+    onStatsCommand(player) { 
+        this.manager_.showStats(player);
```

Comment:
Wouldn't it be much cleaner for the messaging code to live here, and the manager to simply share a structure with the necessary information?

This could also be a great candidate for a dialog.

Reply:
Ah, you want to automatically display it after the match. Hmm.

Reply:
I was still not entirely sure how I wanted to make the dialog.
I kind of like the idea of having a scoreboard like seen on AAD servers:
https://puu.sh/FUu1R/a857a6f9d3.png

So for now I decided to make it as chat messages what I'll probably revise later on.